core/src/main/java/org/apache/brooklyn/entity/group/DynamicFabric.java

```diff
@@ -55,14 +55,14 @@
     @SetFromFlag("memberSpec")
     ConfigKey<EntitySpec<?>> MEMBER_SPEC = ConfigKeys.newConfigKey(
             new TypeToken<EntitySpec<?>>() {}, 
-            "dynamiccfabric.memberspec", 
+            "dynamicfabric.memberspec", 
```

Comment:
Strictly speaking, we should use the `deprecatedNames` to leave support for the old badly spelt name. However, it's unlikely that anyone is using the misspelt name so no strong feelings.

As an example of the deprecation usage:
```
    public static final ConfigKey<Boolean> START_LATCH = ConfigKeys.builder(Boolean.class)
            .name("latch.start")
            .deprecatedNames("start.latch")
            .description("Latch for blocking start (done post-provisioning for software processes); if non-null will wait for this to resolve (normal use is with '$brooklyn:attributeWhenReady')")
            .build();
```